app/script/conversation/ConversationRepository.js

```diff
@@ -2797,7 +2787,13 @@ z.conversation.ConversationRepository = class ConversationRepository {
             return this._onAddEvent(conversationEntity, eventJson);
         }
       })
-      .then((entityObject = {}) => this._handledConversationEvent(entityObject, eventSource, previouslyArchived));
+      .then((entityObject = {}) => this._handledConversationEvent(entityObject, eventSource, previouslyArchived))
+      .catch(error => {
+        const isMessageNotFound = error.type === z.conversation.ConversationError.TYPE.MESSAGE_NOT_FOUND;
+        if (!isMessageNotFound) {
```

Comment:
Would prefer the `isMessageFound` for a variable name.. instead of the double negative thingy.. but here is a bit trickier I guess, because the message type is `MESSAGE_NOT_FOUND`

Reply:
Let's leave it as is.. used only here anyways.

Reply:
Yeah. I see your point but I tried to stick with the error types in this case.